auditLogMover/package.json

```diff
@@ -20,7 +20,7 @@
     "eslint-plugin-prettier": "^3.1.2",
     "jest": "^26.6.3",
     "serverless": "^1.73.1",
-    "serverless-plugin-typescript": "^1.1.9",
+    "serverless-bundle": "4.0.1",
```

Comment:
Using the latest version of `serverless-bundle`, version `4.2.2` result in this error.

Error: Failed to load config "medikoo" to extend from.
More details here: https://github.com/AnomalyInnovations/serverless-bundle/issues/212

Using version 4.0.1 is the latest version in which that bug doesn't appear